tests/store/api/test_store_api_status_analysis.py

```diff
@@ -162,172 +172,29 @@ def test_one_of_one_sequenced_samples(base_store: Store):
-    if not version:
-        version = disk_store.add_version(application, 1, valid_from=datetime.now(), prices=prices)
-        disk_store.add_commit(version)
-    return version
-
-
-def ensure_customer(disk_store, customer_id="cust_test"):
-    """utility function to return existing or create customer for tests"""
-    customer_group = disk_store.customer_group("dummy_group")
-    if not customer_group:
-        customer_group = disk_store.add_customer_group("dummy_group", "dummy group")
-
-        customer = disk_store.add_customer(
-            internal_id=customer_id,
-            name="Test Customer",
-            scout_access=False,
-            customer_group=customer_group,
-            invoice_address="dummy_address",
-            invoice_reference="dummy_reference",
-        )
-        disk_store.add_commit(customer)
-    customer = disk_store.customer(customer_id)
-    return customer
-
-
-def add_sample(
-    store,
-    sample_name="sample_test",
-    received=False,
-    prepared=False,
-    sequenced=False,
-    delivered=False,
-    invoiced=False,
-    external=None,
-):
-    """utility function to add a sample to use in tests"""
-    customer = ensure_customer(store)
-    application_version_id = ensure_application_version(store).id
-    sample = store.add_sample(name=sample_name, sex="unknown")
-    sample.application_version_id = application_version_id
-    sample.customer = customer
-    if received:
-        sample.received_at = datetime.now()
-    if prepared:
-        sample.prepared_at = datetime.now()
-    if sequenced:
-        sample.sequenced_at = datetime.now()
-    if delivered:
-        sample.delivered_at = datetime.now()
-    if invoiced:
-        invoice = store.add_invoice(customer)
-        sample.invoice = invoice
-    if external:
-        sample.is_external = external
-
-    store.add_commit(sample)
-    return sample
-
-
-def add_samples(base_store, n_samples, sequenced):
-    """utility function to add many samples to use in tests"""
-    samples = []
-    for _ in range(n_samples):
-        samples.append(add_sample(base_store, sequenced=sequenced))
-    return samples
-
-
 def relate_samples(base_store, family, samples):
     """utility function to relate many samples to one family"""
 
     for sample in samples:
         base_store.relate_sample(family, sample, "unknown")
 
 
-def add_family_with_samples(base_store, family_name, n_samples, sequenced):
+def add_family_with_samples(base_store, helpers, family_name, n_samples, sequenced_at):
     """utility function to add one family with many samples to use in tests"""
 
-    test_samples = add_samples(base_store, n_samples, sequenced=sequenced)
-    test_family = add_family(base_store, family_name)
+    test_samples = helpers.add_samples(base_store, n_samples)
+    for sample in test_samples:
+        sample.sequenced_at = sequenced_at
+    test_family = helpers.add_case(base_store, family_name)
```

Comment:
```suggestion
    test_case = helpers.add_case(base_store, case_name)
```

Reply:
cahnge family to case also for the other tests

Reply:
fixed